src/chains/models.py

```diff
@@ -18,11 +18,18 @@
 
 
 class Chain(models.Model):
+    class RpcAuthentication(models.TextChoices):
+        API_KEY_PATH = "API_KEY_PATH"
+        NO_AUTHENTICATION = "NO_AUTHENTICATION"
+
```

Comment:
@jpalvarezl what's our standard for exposing multiple words as a value (for type association)?